README.md

```diff
@@ -63,10 +48,6 @@ An opinioned coding guideline for React + Redux project @WORD UP
     * for example: `//TODO: (From Mark) hookup Ajax to rest server`
   
 ## React
-#### Add `propTypes` to every components
-  * It helps others to use the component easily and also help to catch some errors quickly.
-  * In alphabetic order
```

Comment:
I think order by alphabetic is not necessary.
On the other hand, we should try to reduce props number and use better naming for easier to understand.

Reply:
Then how about adding `reduce props number and use better naming for easier to understand` as a rule. Like `refrain from adding more than 5 props which means this component may be too big`. So this is not a hard rule, may stlil worthy to mention it out.

Reply:
agree, update later

Reply:
yeah ordering alphabetically is too much of a development pain. Limiting the amount of props to a component is better

Reply:
@foodaka haha, just about the `a development pain` thing, you can use [VS code pluging](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines)